website/docs/docs/on-premises/releases/1.26.0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 1.26.0
+---
+
+## Release date
+
+07 August 2023
+
+## Features
+
+* Support application/problem+json error response format for all non-200 responses
+* Major version upgrade of AJV from v6 to v8 and Improve support for allOf, anyOf, oneOf, including polymorphic inheritance
+* Add new SCIM role to be assigned to the System Account that is used to access the PactFlow SCIM API.
+
+## Fixes
+
+* Fix error occuring when can-i-deploy badge is requested and no version is found
+* incorrect decoding of pact v4 bodies
+* OAS now supports `exclusiveMinimum` and `exclusiveMaximum` correctly in request and response bodies
+* Fix null pointer in role users endpoint 
+
+## Migration notes
+
+### SCIM role migration
+
+These migration steps are only relevant if the PactFlow SCIM API is in use.
+
+1. Identify the System Account that is being used to access the SCIM API.
+  * Login to PactFlow as an Administrator.
+  * Click the Settings icon in the top right.
+  * Click the `Users` link in the left menu.
+  * Select `System Accounts` in the drop down box above the User list.
+  * Identify the relevant System Account and note the name.
+2. Ensure that no role called `SCIM` exists.
```

Comment:
I thought the migration was updated that made this step unnecessary?

Reply:
Yeah, there is, but I thought it would be better if they did it themselves, because then they know where it's been renamed to. But we could update the docs to say "if you had any role named... it will have been renamed to..."

Reply:
@bethesque mentioned we need to include this bit happy to remove it retroactively if it's not actually required anymore.

Reply:
OK got it, thanks. Let's leave it as is then.